packages/node_modules/@ciscospark/react-component-widget-header/src/index.js

```diff
@@ -12,31 +12,29 @@ import styles from './styles.css';
 const propTypes = {
   onClose: PropTypes.func.isRequired,
   onMenuClick: PropTypes.func.isRequired,
-  participantsCount: PropTypes.number
+  title: PropTypes.string
 };
 
 const defaultProps = {
-  participantsCount: 0
+  title: ''
 };
 
-function RosterHeader({
+function WidgetHeader({
```

Comment:
Is `WidgetHeader` the appropriate name here? When I think `WidgetHeader` I think of the title bar that has the Space Name or the name of the person I'm talking to. What do you think about `ActivityHeader` or `ActivityMenuHeader`?

Reply:
Good call.